src/gluonnlp/model/train/language_model.py

```diff
@@ -15,14 +15,15 @@
 # specific language governing permissions and limitations
 # under the License.
 """Language models for training."""
-__all__ = ['AWDRNN', 'StandardRNN', 'BigRNN']
+__all__ = ['AWDRNN', 'StandardRNN', 'BigRNN', 'ParallelBigRNN']
```

Comment:
Actually `ParallelTransformer` is not exported as part of the API (ie. not part of the respective `__all__`). I think @eric-haibin-lin did that intentionally as we're not fully committed to that API?
If so, the same considerations would apply to `ParallelBigRNN`.
@eric-haibin-lin please recommend if either none or both classes should be added to the API (listed in `__all__`).

Reply:
I don't think it's a good idea to have this as an API. How are you reusing this in estimator?

Reply:
Just like `ParallelTransformer` here https://github.com/liuzh91/gluon-nlp/blob/machine_translation_estimator/src/gluonnlp/estimator/machine_translation_batch_processor.py#L44, I need to define the parallel model in the `BatchProcessor` API. If the `ParallelBigRNN` is in the script, I cannot reuse this class.  

I was aware that Parallel model is a temporary workaround for model parallelization and it will be removed in the future. So maybe we keep both parallel models in the api for now and remove these two parallel models later?

Reply:
@liuzh91 If you don't list the `ParallelBigRNN`, it won't be listed in the online documentation at gluon-nlp.mxnet.io/
But you can still import it for example in https://github.com/liuzh91/gluon-nlp/blob/machine_translation_estimator/src/gluonnlp/estimator/machine_translation_batch_processor.py

Reply:
I see.